tang-operator/tang_operator_test.sh

```diff
@@ -33,19 +33,19 @@ then
     # Architecture unsupported, just let it go in this arch
     echo "Architecture execution bypassed"
     ./runnotest.sh
-    let result=$?
+    result=$?
 elif [ ${dep_res} -eq 0 ];
 then
-    chown minikube.minikube -R $(pwd)
+    chown minikube.minikube -R "$(pwd)"
     # Ugly, but the way paths are managed (root dir not directory where running):
     # /var/tmp/tmt/run-019/plans/default/discover/default/tests/tang/Sanity/tang-operator ========
     # /var/tmp/tmt/run-019/plans/default/execute/data/tang/Sanity/tang-operator/journal.txt: Permission denied
-    chown minikube.minikube -R $(pwd)/../../../../../..
+    chown minikube.minikube -R "$(pwd)"/../../../../../..
```

Comment:
shouldn't it be all quoted?
```suggestion
    chown minikube.minikube -R "$(pwd)/../../../../../.."
```